filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -69,6 +69,10 @@ class OpenGLDriver final : public backend::DriverBase {
         backend::BufferUsage usage = {};
     };
 
+    struct GLBufferObject : public backend::HwBufferObject {
+        GLuint id = 0;
```

Comment:
The convention I have used elsewhere is that all OpenGL state variables go into an anonymous `struct { } gl;` structure.